kedro-org/kedro-viz
# Fixes design issues in metadata panel

## Description

Resolves #1827 

## Development notes

- Made changes to styles based on the comments in #1827
- Updated `Expand Preview` text to be consistent for different dataset types

## QA notes

- Verify fixes using a screen ruler or box model 
- All tests should pass

**NOTE:** Lint issue will be resolved once https://github.com/kedro-org/kedro-viz/pull/2029 is merged

## Checklist
